index.php

```diff
@@ -1,4 +1,12 @@
-<?php require_once __DIR__ . '/vendor/autoload.php'; ?>
+<?php require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
could we move this onto a different line to the opening <? php tag?

Reply:
done